decidim-generators/lib/decidim/generators/app_templates/secrets.yml.erb

```diff
@@ -132,6 +132,8 @@ default: &default
       api_key: <%%= ENV["OMNIAUTH_TWITTER_API_KEY"] %>
       api_secret: <%%= ENV["OMNIAUTH_TWITTER_API_SECRET"] %>
       icon_path: "media/images/twitter-x.svg"
+  api:
+    secret_key_jwt: <%%= ENV["API_SECRET_KEY_JWT"] %>
```

Comment:
These settings should be under `decidim.api` and should not be repeated for every environment.

Reply:
Also, I think `SECRET_KEY` is misleading name. It should be `DECIDM_API_JWT_SECRET`.